bin/pyang

```diff
@@ -438,7 +445,19 @@ Validates the YANG module in <filename> (or stdin), and all its dependencies."""
             kind = "error"
             exit_code = 1
         emsg = etag if o.print_error_code else error.err_to_str(etag, eargs)
-        sys.stderr.write('%s: %s: %s\n' % (epos, kind, emsg))
+
+        if o.msg_template is not None:
+            try:
+                sys.stderr.write(str(o.msg_template).format(file=epos.ref,
```

Comment:
I think you should also add 'kind' (which is 'error' or 'warning')

Reply:
Thanks. I added it as placeholder "type"